python/tools/build_tx.py

```diff
@@ -94,7 +97,7 @@ def _get_inputs_interactive(blockbook_url):
             echo(f"From address: {from_address}")
         except Exception:
             pass
-        amount = tx.bin_outputs[prev_index].amount
+        amount: int = tx.bin_outputs[prev_index].amount
```

Comment:
`pyright` in basic mode was not complaining about it, but strict mode said it was not properly known

Reply:
ughhh that's because of the `List` -> `Sequence` change in messages.py

two ways to solve this correctly: (a) in messages.py, output `self.bin_outputs: Sequence["TxOutputBinType"] = ...`
(b) revert back to `List`

what were the consequences of (b) again? we had to call `list()` on something, iirc.

i'm a little torn here, but keeping the `Sequence` type is a tiny bit more correct. (the disadvantage being, code like `tx.bin_outputs.append(...)` would be impossible -- but I don't think we do that anywhere.)

Reply:
The (b) change commit is https://github.com/trezor/trezor-firmware/pull/1893/commits/28f32e0ca3562bbf4c618022fbd474c5377549ea and the changes needed because of that are https://github.com/trezor/trezor-firmware/pull/1893/commits/4365425bc859cf9f23acfde2160c977d2bbf2489

Reply:
thanks.
let's go with (a), please add the type annotation to the generated list items in messages.py

Reply:
Done in ac8afe03e